src/variables.jl

```diff
@@ -203,7 +226,10 @@ function JuMP.set_start_value(
     vref::LogicalVariableRef, 
     value::Union{Nothing, Bool}
     )
-    new_var = LogicalVariable(JuMP.fix_value(vref), value)
+    if has_logical_compliment(vref)
+        error("Cannot set the start value of a variable with a logical compliment.")
```

Comment:
yet?

Reply:
This is to avoid conflicts. I clarified the message.